main.js

```diff
@@ -528,6 +539,13 @@ for(i=0; i<window.cdata.length; i++){
 }
 new Awesomplete(grab("course-list"), {list: j});
 
+/* the awesomplete list for the teachers */
```

Comment:
Lack of instructor names has a purpose: instructor names make the course data >100KB, which we don't want, because then we would need to send it in a second request like doldur.

More importantly, I don't see any use in adding all courses taught by an instructor. For instance, İbrahim Ünal can give Calculus I and Point Topology in the same term, but no one would take these courses at the same time.

Since this PR is about calendar export, you should open a new PR for unrelated changes. You can do that by pushing to a new Git branch in your repository and opening a PR from that branch.

Reply:
I will look into separating the PRs when I'm available. Thanks for the thorough feedback.